resources/nginx/default.conf

```diff
@@ -9,6 +9,9 @@ server {
     location / {
         add_header Last-Modified $date_gmt;
         add_header Cache-Control 'no-store, no-cache, must-revalidate, proxy-revalidate, max-age=0';
+        if ($http_X_Authorized_User) {
+            add_header Request-X-Authorized-User $http_X_Authorized_User;
```

Comment:
WDYT abour renaming to X-Request-Authorized-User? (starting with X- instead)

Reply:
This was to mirror what we do on other probes, where we prepend `Request-` to all headers, while using `X-` could mean it's an actual response header